src/ui/public/vis/vis_types/vislib_vis_legend.html

```diff
@@ -3,7 +3,7 @@
     type="button"
     ng-click="toggleLegend()"
     class="kuiCollapseButton legend-collapse-button"
-    aria-label="Toggle legend"
+    aria-label="{{::'common.ui.vis.visTypes.legend.toggleLegendButtonAriaLabel' | i18n: { defaultMessage: 'Toggle legend' } }}"
```

Comment:
ID: `visualize.vislib.legend.toggleLegendButtonAriaLabel`, please also use `visualize.vislib.legend`, instead of `common.ui.vis.visTypes.legend` for the rest of the i18n ids in this file.

Reply:
please see my comment below